ktor-core/src/org/jetbrains/ktor/http/Dates.kt

```diff
@@ -5,14 +5,24 @@ import java.util.*
 
 fun Long.toHttpDateString(): String {
     val calendar = Calendar.getInstance();
-    calendar.setTimeInMillis(this)
-    val dateFormat = SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss z", Locale.US);
-    dateFormat.setTimeZone(TimeZone.getTimeZone("GMT"));
-    return dateFormat.format(calendar.getTime());
+    calendar.timeInMillis = this
+    return calendar.time.format(httpDateFormat)
 }
+fun Date.toHttpDateString(): String = format(httpDateFormat)
+fun String.fromHttpDateString(): Long = httpDateFormat.parse(this).time
```

Comment:
Then we need both in spite of `java.util.Date` could be easily converted to `java.time` as there are many libraries still use `Date`